Kudu.Core/Jobs/JobsManagerBase.cs

```diff
@@ -84,7 +84,19 @@ protected JobsManagerBase(ITraceFactory traceFactory, IEnvironment environment,
 
             JobsBinariesPath = Path.Combine(Environment.JobsBinariesPath, jobsTypePath);
             JobsDataPath = Path.Combine(Environment.JobsDataPath, jobsTypePath);
-            JobsWatcher = new JobsFileWatcher(JobsBinariesPath, OnJobChanged, null, ListJobNames, traceFactory, analytics, jobsTypePath);
+
+            var appDataPath = Path.Combine(Environment.WebRootPath, Constants.AppDataPath);
+
+            if (JobsBinariesPath.StartsWith(appDataPath, StringComparison.OrdinalIgnoreCase))
```

Comment:
When would it ever not be under App_Data, since jobs are always there? e.g. `wwwroot\jobs\continuous` and `wwwroot\jobs\triggered`. It would be best to avoid two different code paths.

Reply:
I didn't know jobs folder should always under App_Data. I will merge the code.